File: FizzBuzz.lua
local x = 15

for i=1, x, 1 do
local resp = ""
if i % 3 == 0 then
   resp = resp .. "Fizz"
  end
if i % 5 == 0 then 
   resp = resp .."Buzz"
  end
if (i % 5 ~= 0 and i % 3 ~= 0) then
   resp = i
end
print(resp)
end

File: FizzBuzz.kt
object FizzBuzz {
    @JvmStatic
    fun main(args: Array<String>) {
        val x = 15
        for (i in 1 until x + 1) {
            var resp = ""
            if (i % 3 == 0) {
                resp += "Fizz"
            }
            if (i % 5 == 0) {
                resp += "Buzz"
            }
            if (i % 3 != 0 && i % 5 != 0) {
                resp = i.toString()
            }
            println(resp)
        }
    }
}
